ground/src/main/java/com/google/android/ground/ui/datacollection/components/ButtonAction.kt

```diff
@@ -37,7 +37,10 @@ enum class ButtonAction(
 
   // Draw a polygon task
   ADD_POINT(Type.TEXT, Theme.OUTLINED, textId = R.string.add_point),
-  COMPLETE(Type.TEXT, Theme.LIGHT_GREEN, textId = R.string.complete_polygon);
+  COMPLETE(Type.TEXT, Theme.LIGHT_GREEN, textId = R.string.complete_polygon),
+
+  // GPS location task
```

Comment:
Please update comment to new name as well

Reply:
Done